docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,325 @@
 # Registering a Workload Execution Cluster
 
-This document will tell users what how to register a WEC in an ITS.
+This document explains how to register a Workload Execution Cluster (WEC) with an Inventory and Transport Space (ITS) in KubeStellar.
 
-TODO: write this.
+## Overview
+
+Registering a WEC in an ITS is the same process as registering a managed cluster with an OCM hub cluster. KubeStellar uses Open Cluster Management (OCM) for cluster registration and management.
+
+**Terminology Mapping:**
+- **OCM Hub** → **KubeStellar ITS** (Inventory and Transport Space)
+- **OCM Managed Cluster** → **KubeStellar WEC** (Workload Execution Cluster)
+- **OCM Agent** → **OCM Agent** (same component, installed on WEC)
+
+For the complete OCM registration process, refer to the [official Open Cluster Management documentation](https://open-cluster-management.io/docs/getting-started/installation/register-a-cluster/).
+
+## Prerequisites
+
+Before registering a WEC, ensure you have:
+
+1. **A running Kubernetes cluster** that will serve as the WEC
+2. **Network connectivity** between the WEC and the ITS
+3. **`kubectl` access** to both the WEC and ITS
+4. **`clusteradm` CLI tool** installed ([installation guide](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/))
+5. **An existing ITS** with OCM cluster manager running
+6. **Sufficient permissions** to create resources in both clusters
+
+To verify your ITS is ready for WEC registration:
+
+```shell
+# Check if the ITS is accessible (this returns a short result)
+kubectl --context <its-context> get ServiceAccount default
+
+# Verify OCM cluster manager is running (look for pods in Running state)
+kubectl --context <its-context> get pods -n open-cluster-management-hub
+
+# Check for the customization-properties namespace
+kubectl --context <its-context> get ns customization-properties
+```
+
+## Registration Process
```

Comment:
This is the same information as is in https://docs.kubestellar.io/unreleased-development/direct/get-started/#create-and-register-two-workload-execution-clusters . Rather than maintain two copies, we should maintain one copy and refer to it from elsewhere as needed.